java/src/main/java/ai/rapids/cudf/CudfColumnOverflowException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.rapids.cudf;
+
+/**
+ * Exception thrown when CUDF operation results in a column size
+ * exceeding CUDF column size limits
+ */
+public class CudfColumnOverflowException extends CudfException {
```

Comment:
I feel that "ColumnOverflow" is not a good name. Maybe `....SizeOverflow....`?

Reply:
Right. I was thinking that also `ColumnSizeOverflow`.

Reply:
I have renamed this to `CudfColumnSizeOverflowException`. It's a mouthful, but it's more accurate.